lib/v2/ustb/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/yjsy/news/:type?', require('./yjsy/news'));
```

Comment:
```suggestion
    router.get('/yjsy/news/:type', require('./yjsy/news'));
```
It's not an optional parameter according to your docs.

Reply:
thank u